libs/automation/nx.json

```diff
@@ -1,15 +1,5 @@
 {
   "$schema": "./node_modules/nx/schemas/nx-schema.json",
-  "tasksRunnerOptions": {
-    "default": {
-      "runner": "@nrwl/nx-cloud",
-      "options": {
-        "cacheableOperations": ["build", "test", "lint", "package", "prepare"],
-        "canTrackAnalytics": false,
-        "accessToken": "ZTQ3Yzc3MjEtM2M2ZS00MzRlLWI0OWItZjhmOWVhNWQ1MTM0fHJlYWQ="
```

Comment:
how we will reuse the Nx cloud cache?

Reply:
According to the latest NX docs, this setting is not necessary anymore. Let me double-check that.